oak/server/rust/oak_runtime/src/runtime/tests.rs

```diff
@@ -140,3 +140,68 @@ fn create_node_more_public_label() {
         }),
     );
 }
+
+#[test]
+fn wait_on_channels_immediately_returns_if_any_channel_is_orphaned() {
+    run_node_body(
+        Label::public_trusted(),
+        Box::new(|runtime| {
+            // Create channels with handles
+            let (write_handle, read_handle_1) = runtime.channel_create(&Label::public_trusted());
+            let (_write_handle, read_handle_2) = runtime.channel_create(&Label::public_trusted());
+
+            // Close the write_handle; this should make the channel Orphaned
+            let result = runtime.clone().channel_close(write_handle);
+            assert_eq!(Ok(()), result);
+
+            // Wait on channel
```

Comment:
Here there are two channels. Also I don't think this comment is particularly relevant.

Reply:
Oh I just meant that maybe it should be channels (plural). But also that it also seems somewhat obvious (usually comments are for "why", not "what"), but feel free to keep it if you think it helps explain the flow.